DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: hector
 Title: The Hector Simple Climate Model
-Version: 2.3.0
+Version: 2.3.1
```

Comment:
```suggestion
Version: 2.4.0
```

The version string will also need to be changed in `h_util.hpp`.

Reply:
Resolved in 1f4bbc4.